From 99d8c7dded382d9fa2da95e6a2984c760ef25b04 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Mon, 7 Jul 2025 16:44:24 +0200
Subject: [PATCH] Makefile: drop absl_bad_optional_access

It's no longer provided by newer abseil and doesn't seem to be needed.

Signed-off-by: Martin Jansa <martin2.jansa@lgepartner.com>
---
Upstream-Status: Pending

 makefile_build/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/makefile_build/Makefile b/makefile_build/Makefile
index 9044b93..7808d64 100644
--- a/makefile_build/Makefile
+++ b/makefile_build/Makefile
@@ -37,7 +37,6 @@ LIBEDGETPU_LDFLAGS := \
 	-labsl_hashtablez_sampler \
 	-labsl_raw_hash_set \
 	-labsl_str_format_internal \
-	-labsl_bad_optional_access \
 	-lusb-1.0
 
 LIBEDGETPU_FLATC_SRCS := \
